pkg/installer/installation/reset.go

```diff
@@ -51,14 +55,21 @@ func Reset(ctx *util.Context) error {
 func destroyWorkers(ctx *util.Context) error {
 	ctx.Logger.Infoln("Destroying worker nodes…")
 
-	if err := util.BuildKubernetesClientset(ctx); err != nil {
-		return errors.Wrap(err, "unable to build kubernetes clientset")
-	}
-	if err := machinecontroller.DestroyWorkers(ctx); err != nil {
-		return errors.Wrap(err, "unable to delete all worker nodes")
+	waitErr := wait.ExponentialBackoff(defaultRetryBackoff(3), func() (bool, error) {
+		err := util.BuildKubernetesClientset(ctx)
+		return err == nil, errors.Wrap(err, "unable to build kubernetes clientset")
+	})
+	if waitErr != nil {
+		ctx.Logger.Info("try to use `--destroy-workers=false` flag")
```

Comment:
I'd like some context here why it failed if we don't have it already. Something like
```
We are unable to connect to the control plane and destroy worker nodes. You can skip destorying nodes and destroy them manually using `--destroy-workers=false`
```

Reply:
👍